docs/kubernetes-installers.md

```diff
@@ -37,6 +37,29 @@ want. Once the cluster has been created, this CIDR cannot be changed.
 
 ### Rancher
 
+Follow these [steps](https://rancher.com/docs/rancher/v1.6/en/installing-rancher/installing-server/) to deploy Rancher server UI dashboard.
+
+* Sign in to the UI dashboard, and switch to cluster management.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_cluster_management.png" width="800">
+
+* Create new Node template there under the RKE configuration section.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_node_template.png" width="800">
+
+* After saving the template, proceed with creating new cluster, click on create button under the clusters tab, then enter the name, number of worker and control plane Nodes respectively.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_create.png" width="800">
+
+* click on "Edit as YAML", and set the network plugin to null, and add an addon for Antrea, in the following manner:
+addons_include:
+  -<link of the antrea.yml file>
+
+You can find a few screenshots for validation after following the above steps:
+<img src="https://downloads.antrea.io/static/04062023/rancher_cluster.png" width="800">
+<img src="https://downloads.antrea.io/static/04062023/rancher_node.png" width="800">
+
+Note: Antrea (since v1.10.0) is supported on Rancher version v2.7.0+.
```

Comment:
Again, it's confusing that the linked guide is for v1.6 while we keep talking about v2.7.0.